app/models/post.rb

```diff
@@ -3,15 +3,19 @@ class Post < ApplicationRecord
   has_many :comments, class_name: 'Comment', foreign_key: 'post_id'
   has_many :likes, class_name: 'Like', foreign_key: 'post_id'
 
-  after_save :update_post_counter
-
-  private
-
-  def update_counter
+  def update_posts_counter
     author.increment!(:posts_counter)
   end
 
   def return_comments
     comments.order(created_at: :desc).limit(5)
   end
```

Comment:
- [ ] Kindly add tests for these two callback functions.

![model-methods](https://user-images.githubusercontent.com/53124548/215268416-18912fe5-5419-4c91-ae37-9317024ba56b.png)